src/main/java/com/easyrest/pages/waiterPanel/InProgressOrders.java

```diff
@@ -0,0 +1,12 @@
+package com.easyrest.pages.waiterPanel;
+
+import com.easyrest.pages.WaiterPage;
+import org.openqa.selenium.WebDriver;
+
+public class InProgressOrders extends WaiterPage {
```

Comment:
Inheritance is a "Is-A" relation. In your case Waiter page only contains InProgressOrders so you have "Has-A' relation